README.md

```diff
@@ -1,5 +1,7 @@
 # Northcoders News API
 
+**you need to create a jestEnvVar.js with all of the env variables so the tests can run properly**
```

Comment:
Is this not handled by your `.env.test` and `.env.development` files?

Reply:
No it isn't, I need to create this file in order to run the tests because jest is not reading the env files for some reason.